cubelib/src/cube/cube.rs

```diff
@@ -31,6 +32,14 @@ impl Cube333 {
     pub fn new(edges: EdgeCube333, corners: CornerCube333) -> Cube333 {
         Cube333 { edges, corners }
     }
+    pub fn random() -> Cube333 {
```

Comment:
The random cube generator. Decided to pass the rng as an argument instead of creating it inside the edge/corner's `random` methods in case there's some correlation from initializing two rng's right next to each other.